scipy/stats/tests/meson.build

```diff
@@ -0,0 +1,27 @@
+py3.install_sources(['test_bootstrap.py',
+    '__init__.py',
+    'test_tukeylambda_stats.py',
+    'test_continuous_basic.py',
+    'test_distributions.py',
+    'studentized_range_mpmath_ref.py',
+    'test_crosstab.py',
+    'test_entropy.py',
+    'test_binned_statistic.py',
+    'test_qmc.py',
+    'test_mstats_basic.py',
+    'test_discrete_distns.py',
+    'test_hypotests.py',
+    'test_stats.py',
+    'test_contingency.py',
+    'test_mstats_extras.py',
+    'test_rank.py',
+    'test_kdeoth.py',
+    'common_tests.py',
+    'test_multivariate.py',
+    'test_morestats.py',
+    'test_discrete_basic.py',
+    'test_fit.py',
+    'test_relative_risk.py'],
+    pure : false,
+    subdir : 'scipy/stats/tests'
+)
```

Comment:
There's a `subdir('data')` missing, and a few more `subdir`'s below that.

Reply:
This kind of thing will be easy to catch only the tests run (which require `special`, `spatial`, and maybe more).